src/proj_4D_api.c

```diff
@@ -858,6 +823,9 @@ PJ_FACTORS proj_factors(PJ *P, LP lp) {
     PJ_FACTORS factors = {0,0,0, 0,0,0, 0,0};
     struct FACTORS f;
 
+    if (0==P)
+        return factors;
```

Comment:
Return type is  `PJ_INIT_INFO` but factors is `PJ_FACTORS`.  Confusing.

Reply:
No - there is a new function, `proj_factors` starting in between